app/controllers/users_controller.rb

```diff
@@ -47,6 +49,17 @@ def create
       end
       # send all errors to the page so the user can try again
       @action = "create"
+      email_errors = @user.errors.messages[:email]
```

Comment:
can we shift `is invalid` to other PR

Reply:
Okay, I will update remove it from this PR